src/PIL/JpegImagePlugin.py

```diff
@@ -25,6 +25,7 @@
 # 2009-09-06 fl   Added icc_profile support (from Florian Hoech)
 # 2009-03-06 fl   Changed CMYK handling; always use Adobe polarity (0.6)
 # 2009-03-08 fl   Added subsampling support (from Justin Huff).
+# 2019-07-05 fl   Use Adobe APP14 marker ColorTransform tag.
```

Comment:
My presumption is that 'fl' here stands for 'Fredrik Lundh', as seen a few lines later in the copyright.

Reply:
Sorry, I didn't notice the copyright aspect. My intention was simply to add an item to the history list.